.github/workflows/unit-tests.yml

```diff
@@ -40,5 +50,10 @@ jobs:
           # need this to also fetch tags
           fetch-depth: 0
 
+      - name: Build unit test container if it changed
```

Comment:
Can we DRY this a bit somehow?

Reply:
I added a new commit to make the build command shorter, which helps a bit already.

Beyond that, it would be nice to factorize/parameterize the whole `steps:` or even the whole `job` entry. I have not yet learned about [github workflow custom actions](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions) to do that. I want to do that when introducing automatic unit-test container refresh with gating, as that will have to run through similar steps -- but it will be *more* code in total, and thus also be harder to understand what happens at a glance, so I'd like to not commit any of that just yet.

So, I know that this is repetitive, but IMHO it doesn't hurt too much for now.